packages/api-browser/bundle.js

```diff
@@ -0,0 +1,13 @@
+// Need to import using this syntax as require doesn't work for local files + rollup
+import * as browserApi from './build/es/index.js';
```

Comment:
Yeah you're right, I never thought of that, was trying to reproduce the bundle in this packages, but I don't think it's necessary :-)